Readme.md

```diff
@@ -29,6 +29,8 @@ Sweet.
 
 It'll work just like a property: when the instance is deinit'd, the `DisposeBag` gets disposed. It's also a read/write property, so you can use your own, too.
 
+If you want to add a DisposeBag to an Object that does not inherit from NSObject, you can also implement the protocol "DisposeBagable", and you're good to go. This protocol provides a default DisposeBag called rxDisposeBag.
```

Comment:
Excellent addition to the readme 👍

Reply:
Wondering on why we should call this `rxDisposebag` given the fact it's from RxSwift is rather implicit? what about just calling it disposeBag across the board?

Reply:
Good point. I will fix that.